### ISSUE TITLE
Problem to run dev

### ISSUE THREAD

**Issue by farnetani** [MajesticPotatoe/electron-vuetify#1]
Hi, I have a problem below when I executed the command: 

```
npm run dev
```

Problem:

```
Uncaught TypeError: fs.existsSync is not a function
    at getElectronPath (index.js?bdb9:7)
    at Object.eval (index.js?bdb9:18)
    at eval (index.js:20)
    at Object../node_modules/electron/index.js (app.js:2099)
    at __webpack_require__ (app.js:770)
    at fn (app.js:130)
    at eval (window.js?bbec:1)
    at Module../src/store/window.js (app.js:7768)
    at __webpack_require__ (app.js:770)
    at fn (app.js:130)
```

**Comment by MajesticPotatoe** [comment]
I can take a look later tonight. I generally dont use npm, mainly use yarn. Though there could have been a minor update since I last touched thise that is causeing the issue.

**Comment by MajesticPotatoe** [comment]
As it stands right now, `npm run dev` will never work with the current window store. This repo is very much so in its infancy and I have yet to decide how I would like to deal with this as its specific to `npm run electron-dev`.

**Comment by MajesticPotatoe** [comment]
Happy to let you know this has been resolved in 71735aa.
